### PR TITLE
styleguide v2

### PR DESCRIPTION
**Pull request by cindywu**
**v2**

- [x] define black color for text
- [x] define base and serif text styles
- [x] define brand colors and removes excess colors

**v3**
- [ ] define button styles
- [ ] define link styles
- [ ] define brand green (current green is too loud)
- [ ] form inputs (normal, active, error)
- [ ] bulleted list
- [ ] bold text
- [ ] checkbox and radio buttons

**vn**

replace h1 with serif font that feels like a heavier and wider cousin of Times, inspired by Tiempos Headline

### CODE REVIEW

File: app/assets/stylesheets/core/_type.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
-html, body, h1, h2, h3, h4, h5,
+html, body, h1, h2, h3, h4, h5, h6
 ul, ol, li, blockquote, label, form,
 input, span, a {
   font-family: 'Roboto', Helvetica, Arial, sans-serif;
-  color: $black;
+  color: $shaleDark;
```
Comment (dennyluan) [inline app/assets/stylesheets/core/_type.scss]: I generally prefer not to use straight black for significant amounts of body text.

File: app/assets/stylesheets/core/_variables.scss [hunk 2]
```diff
@@ -34,14 +34,14 @@ $orangeyRose:              #FEB09B;
 $teal:                     #43B596;
 
 // brand
+$shaleDark:				#212121;
 $shale:           #4a5658;
 $shaleLight:      #9Ba5aa;
 $shaleLighter:    #D4D9DB;
 $shaleLightest:   #F9FBFB;
-$shaleKindaLight: #A5A7A8;
```
Comment (dennyluan) [inline app/assets/stylesheets/core/_variables.scss]: Is there a reason to remove these? I frequently go for these colors and having these shorthands is useful.

File: app/assets/stylesheets/core/_type.scss (hunk 1 above)
Comment (cindywu) [reply app/assets/stylesheets/core/_type.scss]: Did you mean, you prefer to use straight black?

File: app/assets/stylesheets/core/_variables.scss (hunk 2 above)
Comment (cindywu) [reply app/assets/stylesheets/core/_variables.scss]: The reason is the naming is inconsistent. 

If you need ```#A5A7A8``` we can rename it to ```#shaleLighterer``` or wherever it falls on the spectrum.

File: app/assets/stylesheets/core/_type.scss (hunk 1 above)
Comment (dennyluan) [reply app/assets/stylesheets/core/_type.scss]: I prefer to not use black for body text. IMO it's a little bit easier on the eyes for reading long chunks of body text if its a little bit lighter (e.g. https://designforhackers.com/blog/impressionist-color-theory/)

File: app/assets/stylesheets/core/_variables.scss (hunk 2 above)
Comment (dennyluan) [reply app/assets/stylesheets/core/_variables.scss]: shaleKindaLight is darker than shaleLight, so naming it shaleLighterer wouldn't make sense.

File: app/assets/stylesheets/core/_type.scss (hunk 1 above)
Comment (dennyluan) [reply app/assets/stylesheets/core/_type.scss]: Nevermind I am confused.

File: app/assets/stylesheets/core/_variables.scss (hunk 2 above)
Comment (dennyluan) [reply app/assets/stylesheets/core/_variables.scss]: Right now we're not using it anywhere so we could remove it.

**Pull request merged**